src/vector.rs

```diff
@@ -126,12 +127,25 @@ impl Vector {
             other.unit() * self.dot_product(other) / other.magnitude()
         }
     }
+
+    /// Rotate a vector by the given amount (counterclockwise)
+    pub fn rotate(self, rotation: Radians) -> Vector {
+        let (rotation_sin, rotation_cos) = (-rotation.value()).sin_cos();
```

Comment:
Why is the negative of `rotation.value()` taken here? Leave a comment.

Reply:
1a63aa6